omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/dispatch.lua.erb

```diff
@@ -44,9 +44,15 @@ if mode == "api" then
         ngx.exit(ngx.HTTP_OK)
       end
     else
-      -- the request did not originate with a chef client, we'll give something
-      -- more friendly to web browsers.
-      return ngx.exec("/index.html")
+      <% if node['private_chef']['nginx']['show_welcome_page'] %>
```

Comment:
[nit] we could use `-%>` here for less newlines but also it doesn't really matter